### PR TITLE
Game created code improvements

### PR DESCRIPTION
**Pull request by poixen**
+ No longer get "5hrs" and "5+ hrs" instead will be "4hrs" and "5+ hrs".
+ Refactored into an else-if with a single return.
+ removed comments
+ removed code from namespace and added to class

### CODE REVIEW

**Comment by ctrlaltca** [comment]
:+1:

**Comment by ZeldaZach** [comment]
Is it possible we can keep minute count all the way up until we hit the 5+ hour mark? 

(Ex: 1hr 49m; 2h 18m; 3h 57m)

**Comment by Daenyth** [comment]
I don't think that's valuable. It's less cognitive load to bucket the times

**Comment by Daenyth** [comment]
:shipit:

File: cockatrice/src/gamesmodel.h [hunk 1]
```diff
@@ -18,6 +18,8 @@ class GamesModel : public QAbstractTableModel {
     QMap<int, GameTypeMap> gameTypes;
 
     static const int NUM_COLS = 8;
+    static const int SECS_PER_MIN  = 60;
```
Comment (ZeldaZach) [inline cockatrice/src/gamesmodel.h]: I spy **two spaces** between the variable name and the equals sign :baby_chick:

**Pull request merged**